### DISCUSSION TITLE
Alguien más a resultado víctima de usuarios de esta página

### DISCUSSION THREAD

**Discussion by alexpc90** [community/community#69598] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Disculpen las molestias pero no sé si soy la única persona o existen más con algún problema similar, en alguna ocasión en esta página descarge algún programa y de alguna forma no sé si por ese punto o por algún otro estuve siendo acosado intimidado y extorcinado por algún grupo con el tiempo me percate que muchas páginas están vinculadas al sitio y se cálamo hasta que destruyeron mi cpu y dejaron sin funcionamiento varios dispositivos mas, al final fue más un daño más psicológico que material pero de alguna manera arruinaron mi fuente de ingresos y no sé si solo fui una víctima a la asar o es avitual.

**Comment by moayad-star** [comment]
Lamento escuchar que has tenido una experiencia tan negativa en línea. Lo que mencionas suena muy grave y es importante abordarlo adecuadamente. Si crees que has sido víctima de acoso, intimidación o extorsión en línea, te recomiendo que tomes las siguientes medidas:

1. Denuncia el incidente a las autoridades locales o a una unidad de ciberdelitos, si es posible. Proporciona toda la información relevante que tengas.

2. Cambia tus contraseñas de inmediato y asegúrate de que tus cuentas en línea estén protegidas con contraseñas fuertes y únicas.

3. Actualiza tus programas de seguridad y antivirus en tus dispositivos.

4. Evita hacer clic en enlaces sospechosos o descargar archivos de fuentes no confiables.

5. Limita la cantidad de información personal que compartes en línea y revisa la configuración de privacidad en tus cuentas de redes sociales.

6. Mantén un registro de cualquier evidencia que puedas tener de los acosadores, como capturas de pantalla, correos electrónicos o mensajes.

7. Considera hablar con un profesional de salud mental para obtener apoyo en caso de que estés experimentando estrés psicológico debido a la situación.

Recuerda que es importante tomar medidas para protegerte en línea y denunciar cualquier actividad ilegal a las autoridades. El acoso y la extorsión en línea son delitos graves, y las autoridades pueden ayudarte a investigar y tomar medidas legales contra los responsables.

**Comment by alexpc90** [comment]
Gracias por los consejos en su momento intenté denunciar con la autoridad de mi país pero de alguna forma ellos intervenían los correos y hasta mi teléfono que deje de intentar con el tiempo descubría cada cosa extraña no se cuál fue su intención real ya solo afectaban a los trabajos que realizaba cada vez que empezaba un proyecto lo eliminaban hasta que acabaron con mi negocio y ganas de continuar

**Reply by moayad-star** [reply]
Lamento escuchar que tu situación fue tan complicada y que no pudiste obtener la ayuda necesaria de las autoridades en ese momento. Es frustrante cuando te enfrentas a desafíos en línea y las autoridades no pueden proporcionar la asistencia adecuada.

Si te sientes desanimado y has experimentado un impacto significativo en tu vida y negocio, es fundamental buscar apoyo profesional. Considera hablar con un consejero o terapeuta que pueda ayudarte a lidiar con el estrés y las consecuencias psicológicas de la situación.

Además, para protegerte en línea en el futuro, es importante mantener una buena higiene digital y estar al tanto de las prácticas de seguridad en línea. Utiliza contraseñas fuertes, asegúrate de tener software de seguridad actualizado y sé cauteloso con los enlaces y archivos que descargas.

Recuerda que aunque haya sido una experiencia negativa, muchas personas también han logrado superar situaciones similares y encontrar formas de reconstruir sus vidas y negocios. Mantén una actitud positiva y busca el apoyo que necesitas para recuperarte de esta situación.

**Comment by alexpc90** [comment]
Gracias por tus palabras y tienes razón una mala experiencia no es el fin.

**Reply by moayad-star** [reply]
I am happy to hear this
